stl/inc/__msvc_string_view.hpp

```diff
@@ -628,6 +628,21 @@ constexpr size_t _Traits_find(_In_reads_(_Hay_size) const _Traits_ptr_t<_Traits>
         return _Start_at;
     }
 
+#if _USE_STD_VECTOR_ALGORITHMS
+    if constexpr (_Is_implementation_handled_char_traits<_Traits> && sizeof(_Traits::char_type) <= 2) {

```

Comment:
Missing `typename`, also found by the test coverage. I know that the CI was down, but as a reminder, it's a good idea to always locally run edited tests before pushing changes.